Sources/TensorFlow/Core/TensorHandle.swift

```diff
@@ -287,6 +289,7 @@ extension ShapedArray where Scalar: _TensorFlowDataTypeCompatible {
   }
 }
 
+#if !USING_X10_BACKEND
 // Tensor conversion.
 extension Tensor {
   public init(_ array: __owned ShapedArray<Scalar>) {
```

Comment:
Out of curiosity, there should be an alternate implementation of this initializer that doesn't try a dynamic cast to CTensorTensorBuffer?

Reply:
That would be:
     self = array.buffer.withUnsafeBufferPointer { buffer in
        return Tensor(shape: TensorShape(array.shape), scalars: buffer)
      }

It exists in Tensor<>. I can merge them once this is in.